### PR TITLE
Add section for custom templates

### PR DESCRIPTION
**Pull request by Barabazs**

### CODE REVIEW

**Comment by Barabazs** [comment]
Pinging @adem @Ameb @CTXz @edupont @Liblor @lucafluri
Is this repo abandoned?

**Comment by Liblor** [comment]
Hi,

Thanks for your pull request! I think people are just busy, I'll try to have a look at some of the pull requests in the coming days.

Before merging the pull request, could you please do the following:
- add a period (".") to each bullet point.
- sort the entries alphabetically.
- use consistent capitalization (e.g. "Lines Project" could be "lines project")

see also [Contribution Guidelines](https://github.com/reHackable/awesome-reMarkable/blob/master/CONTRIBUTING.md).

It is also suggested to only include one suggestion per pull request, this makes removing entries as easy as a `git revert`. However, personally I'm fine with making an exception. Maybe someone who's worked a bit more on the list has an objection, if not I'll merge it, after the other issues have been addressed.

Again thank you for helping the reMarkable community!

**Comment by Barabazs** [comment]
Thank you for your review @Liblor .
I made the requested changes.

**Review by Liblor — APPROVED** [review]

**Pull request merged**
